app/src/main/res/values/strings.xml

```diff
@@ -386,13 +386,13 @@
 
     <!-- cert4android -->
     <string name="certificate_notification_connection_security">EteSync: Connection security</string>
-    <string name="trust_certificate_unknown_certificate_found">EteSync has encountered an unknown certificate. Do you want to trust it?</string>
+    <string name="trust_certificate_unknown_certificate_found">EteSync has encountered an unknown certificate. Do you want to entrust it?</string>
```

Comment:
While technically more correct, wouldn't users find it confusing/odd? Especially non-native speakers?

Reply:
Well there are translations now, so there is that. In my head "Do you trust it", or "Do you want to entrust it"? Could be wrong, and maybe the former is better? I like that it sounds a bit scary, because it should.

Reply:
On second thought, I'm not even sure entrust is more correct than trust (though I really have no idea). I think though trust makes more sense as it's clearer for most people, and as you said, it does sound scarier (which is good).